kudos.yaml

```diff
@@ -2147,4 +2147,15 @@ name: captain
     - just for fun
     - meme
   image: ShowMeWhatYouGot.svg
+  - name: tamagotchi
+  to_address: 0x4b00a17a8A4c7dC2f71777207B1C1Bd6C132D394
```

Comment:
unfortunately we're not offering to have artists own this batch of kudos.  we're going to have to airdrop them at ethereal, and to do that gitcoin has to own them.